packages/core/src/Site/index.js

```diff
@@ -177,9 +177,9 @@ class Site {
    * @param rootPath
    * @param templatePath
```

Comment:
add jsdoc param here, and i would suggest `isConvert`

Reply:
Just opened another PR #2267 that will remove the need for JSDocs here!